Kickstarter-iOS/Views/Cells/SearchProjectCell.swift

```diff
@@ -65,7 +76,29 @@ internal final class SearchProjectCell: UITableViewCell, ValueCell {
       |> UIView.lens.layoutMargins .~ .init(top: Styles.grid(1), left: 0, bottom: 0, right: 0)
       |> UIView.lens.backgroundColor .~ .clear
 
-    _ = self.separateView
-      |> separatorStyle
+    _ = self.separateView |> separatorStyle
+
+    _ = self.statsStackView |> UIStackView.lens.spacing .~ Styles.grid(1)
+  }
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.projectImageUrl
+      .observeForUI()
+      .on(event: { [weak self] _ in
+        self?.projectImageView.image = nil
+        self?.projectImageView.af_cancelImageRequest()
+      })
+      .skipNil()
+      .observeValues { [weak self] in
+        self?.projectImageView.af_setImage(withURL: $0)
+    }
```

Comment:
we can use the new `imageView.rac.imageUrl` reactive extension here if you are interested...